packetbeat/flows/worker.go

```diff
@@ -154,21 +208,16 @@ func newFlowsWorker(pub Reporter, watcher *procs.ProcessesWatcher, table *flowMe
 		if ticksPeriod == 0 {
 			ticksPeriod = 1
 		}
-	}
-
-	debugf("new flows worker. timeout=%v, period=%v, tick=%v, ticksTO=%v, ticksP=%v",
-		timeout, period, tick, ticksTimeout, ticksPeriod)
 
-	defaultBatchSize := 1024
-	processor := &flowsProcessor{
-		table:    table,
-		watcher:  watcher,
-		counters: counters,
-		timeout:  timeout,
+		if activeTimeout > 0 {
```

Comment:
same question about this block - possible to add a comment describing what ishappening?